src/t_list.c

```diff
@@ -1197,12 +1198,12 @@ void lmpopGenericCommand(client *c, int numkeys_idx, int is_block) {
     }
 }
 
-/* LMPOP numkeys [<key> ...] LEFT|RIGHT [COUNT count] */
+/* LMPOP numkeys <key> [<key> ...] (LEFT|RIGHT) [COUNT count] */
```

Comment:
i think the LEFT|RIGHT one is ok? like https://redis.io/commands/lmpop/   and ZMPOP
```suggestion
/* LMPOP numkeys <key> [<key> ...] LEFT|RIGHT [COUNT count] */
```

Reply:
I prefer to keep consistent with other command comments in same file. If you check the function lmoveCommand, blmoveCommand,  the format of them is (LEFT|RIGHT) instead of LEFT|RIGHT. How do you think?  Thanks